mirrord/layer/src/file/filter.rs

```diff
@@ -100,6 +100,7 @@ fn generate_remote_ro_set() -> RegexSet {
         // AWS cli cache
         // "file not exist" for identity caches (AWS)
         r".aws/cli/cache/.+\.json$",
+        r".aws/credentials$",
```

Comment:
Nitpick but maybe add `.aws/config` too for default region

Reply:
nice one!

Reply:
nice one!